src/pages/register/Register.js

```diff
@@ -50,7 +51,7 @@ function Register() {
         className="registerContainer"
       >
         <label>
-          Email
+          email
           <input
             type="email"
             placeholder="janedoe@gmail.com"
```

Comment:
@Alzhan-B From a UX/UI we can remove placeholders (except for password field) since we are adding labels to input fields.
here is a article that elaborates on this idea [blog](https://www.smashingmagazine.com/2018/10/form-design-patterns-excerpt-a-registration-form/#:~:text=The%20label%20and%20placeholder%20text,to%20give%20users%20a%20hint.)

Reply:
Thanks! Removed placesholders for email and username as per your request.